### ISSUE TITLE
Understanding mixed-use buildings

### ISSUE THREAD

**Issue by AlexJew** [architecture-building-systems/CityEnergyAnalyst#2758]
**Describe the feature you would like to have**
Hello, I would like to understand the difference between stating different uses in _typology.dbf_ and building mixed-uses usetypes in the databases via the **Data Editor**. I believe that if a mixed-use can be define by mixing schedules of different use-types, just refering to different use-types with their respective rates per building in _typology.dbf_ is enough

**Describe the end purpose of this feature**
Higher efficiency

**Describe the end-user of this feature**
Any practioner

**Provide Aditional information if possible**
EX. Screenshots, Links, Methods

**Comment by daren-thomas** [comment]
@AlexJew I agree, the feature in the Database Editor is maybe not named correctly: It's creating a _new_ use_type (based on existing use types) that can then be independently edited.

So, the feature in the Database Editor allows you to create independent use types - that can then be mixed together in the typology input file.

What do you think: Would it be a bit less confusing if we renamed the tool to "Create new use-type"? And then set the name (currently "mixed_use") to be based on the use type shown in the Database Editor when you click "Add new Use-Type", say it was "HOTEL", the form should read:

- name: HOTEL_copy
- metadata: based on HOTEL
- ratios: HOTEL|1.0

**Comment by AlexJew** [comment]
Hi @daren-thomas , I believe it is a good idea indeed. 

1. The title could be "Create new use-type" and for clarity, maybe you could formulate it differently than "Creates a new use-type by aggregating values from a list of different use-types" to "Creates a new use-type using as a basis a single or multiple existing use-types" (or something similar that suggest that indeed it can be built from a single existing use-type)

2. I believe that to begin with I wasn't able to conceptually understand the difference between building a mixed use-type and assigning shares of the GFA to different use-types. However, when allocating different use-types to a single building (via use rates), is there a way to modify or visualise its multiple schedules in the **Input Editor**? I believe that currently only the 1st use-type's schedules are visible and modifiable to the user. 

Thanks for the help :)
